app/controllers/application_controller.rb

```diff
@@ -23,4 +23,18 @@ def append_info_to_payload(payload)
     super
     payload["params"] = request.params
   end
+
+  def redirect_back_or(fallback)
+    if param_location = params[:redirect_to].to_s.presence
+      if param_location.is_a?(String) && param_location.start_with?('/')
+        redirect_to URI("http://nope.nope#{param_location}").request_uri # using URI to silence Brakeman
```

Comment:
Ah, I thought `#request_uri` returned the the host too.  That's a little weird, but makes sense.